lib/gc/Transforms/GPU/Pipeline.cpp

```diff
@@ -76,7 +76,10 @@ void populateGPUPipeline(mlir::OpPassManager &pm) {
   pm.addNestedPass<func::FuncOp>(createGpuMapParallelLoopsPass());
   pm.addNestedPass<func::FuncOp>(createParallelLoopToGpuPass());
 
-  pm.addNestedPass<func::FuncOp>(imex::createInsertGPUAllocsPass("opencl"));
+  imex::InsertGPUAllocsOptions insertGPUAllocsOption{
+      /*clientAPI*/ "opencl", /*inRegions*/ false, /*isUsmArgs*/ true};
```

Comment:
can we make `isUsmArgs` a parameter of the pipeline?

Reply:
Yes! An option is added to the pipeline